theme/static/js/resourceLandingAjax.js

```diff
@@ -1591,23 +1593,86 @@ function setFileTypeSpatialCoverageFormFields(logical_type){
     if (logical_type !== "GenericLogicalFile"){
         // don't allow changing coverage type
         $id_type_filetype_div.parent().closest("div").css('pointer-events', 'none');
-        $id_type_filetype_div.find("#id_type_1").attr('onclick', 'return false');
-        $id_type_filetype_div.find("#id_type_2").attr('onclick', 'return false');
+        $id_type_filetype_div.find(radioBoxSelector).attr('onclick', 'return false');
+        $id_type_filetype_div.find(radioPointSelector).attr('onclick', 'return false');
         if (logical_type !== "RefTimeseriesLogicalFile"){
-            $id_type_filetype_div.find("#id_type_1").attr('checked', 'checked');
+            $id_type_filetype_div.find(radioBoxSelector).attr('checked', 'checked');
+        }
+        $id_type_filetype_div.find(radioPointSelector).attr('disabled', true);
+        $id_type_filetype_div.find(radioPointSelector).parent().closest("label").addClass("text-muted");
+        if (logical_type === "NetCDFLogicalFile" || logical_type === "GeoRasterLogicalFile"){
+            $("#id-spatial-coverage-file-type").attr('data-coordinates-type', 'rectangle');
+            $("#id-spatial-coverage-file-type").coordinatesPicker();
+            $("#id-origcoverage-file-type").attr('data-coordinates-type', 'rectangle');
+            $("#id-origcoverage-file-type").coordinatesPicker();
         }
-        $id_type_filetype_div.find("#id_type_2").attr('disabled', true);
-        $id_type_filetype_div.find("#id_type_2").parent().closest("label").addClass("text-muted");
     }
     else {
-        // file type is "GenericLogicalFile" - allow changing coverage type
-        $id_type_filetype_div.find("input:radio").change(updateEditCoverageState);
+        // file type is "GenericLogicalFile" or "FileSetLogicalFile" - allow changing coverage type
+        $id_type_filetype_div.find("input:radio").change(updateEditCoverageStateFileType);
+        var onSpatialCoverageDelete = function () {
```

Comment:
Again this function is not needed for this fix - it is specific to file set work.